pkg/plugins/policies/meshtimeout/plugin/testdata/basic_inbound_listener.golden.yaml

```diff
@@ -8,6 +8,8 @@ filterChains:
   - name: envoy.filters.network.http_connection_manager
     typedConfig:
       '@type': type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+      commonHttpProtocolOptions:
```

Comment:
A test that shows the removal of the timeout from the old Timeout would be good.

Reply:
That's more difficult to test than I thought initially. We don't call inbound/outbound generators in these plugin tests, that's why old Timeout policy is not applied.